packages/manager/src/mocks/presets/crud/handlers/nodebalancers.ts

```diff
@@ -0,0 +1,774 @@
+        'nodeBalancerConfigs',
+        configId
+      );
+
+      if (!nodeBalancer || !nodeBalancerConfig) {
+        return makeNotFoundResponse();
+      }
+
+      const payload = await request.clone().json();
+
+      const nodeBalancerConfigNode = nodeBalancerConfigNodeFactory.build({
+        ...payload,
+        config_id: configId,
+        nodebalancer_id: nodeBalancerId,
+      });
+
+      await mswDB.add(
+        'nodeBalancerConfigNodes',
+        nodeBalancerConfigNode,
+        mockState
+      );
+
+      queueEvents({
+        event: {
+          action: 'nodebalancer_node_create',
+          entity: {
+            id: nodeBalancerId,
+            label: nodeBalancer.label,
+            type: 'nodebalancer',
+            url: `/v4/nodebalancers/${nodeBalancerId}`,
+          },
+        },
+        mockState,
+        sequence: [{ status: 'notification' }],
+      });
+
+      return makeResponse(nodeBalancerConfigNode);
+    }
+  ),
+];
+
+export const updateNodeBalancer = (mockState: MockState) => [
+  http.put(
+    '*/v4/nodebalancers/:id',
+    async ({
+      params,
+      request,
+    }): Promise<StrictResponse<APIErrorResponse | NodeBalancer>> => {
+      const nodeBalancerId = Number(params.id);
+      const nodeBalancer = await mswDB.get('nodeBalancers', nodeBalancerId);
+      if (!nodeBalancer) {
+        return makeNotFoundResponse();
+      }
+
+      const payload = {
+        ...(await request.clone().json()),
+        updated: DateTime.now().toISO(),
+      };
+
+      const updatedNodeBalancer = { ...nodeBalancer, ...payload };
+      await mswDB.update(
+        'nodeBalancers',
+        nodeBalancerId,
+        updatedNodeBalancer,
+        mockState
+      );
+
+      queueEvents({
+        event: {
+          action: 'nodebalancer_update',
+          entity: {
+            id: nodeBalancerId,
+            label: updatedNodeBalancer.label,
+            type: 'nodebalancer',
+            url: `/v4/nodebalancers/${nodeBalancerId}`,
+          },
+        },
+        mockState,
+        sequence: [{ status: 'notification' }],
+      });
+
+      return makeResponse(updatedNodeBalancer);
+    }
+  ),
+  http.put(
+    '*/v4/nodebalancers/:id/configs/:configId',
+    async ({
+      params,
+      request,
+    }): Promise<StrictResponse<APIErrorResponse | NodeBalancer>> => {
+      const nodeBalancerId = Number(params.id);
+      const configId = Number(params.configId);
+      const nodeBalancer = await mswDB.get('nodeBalancers', nodeBalancerId);
+      const nodeBalancerConfig = await mswDB.get(
+        'nodeBalancerConfigs',
+        configId
+      );
+
+      if (!nodeBalancer || !nodeBalancerConfig) {
+        return makeNotFoundResponse();
+      }
+      const payload = await request.clone().json();
+      delete payload?.nodes;
+
+      const updatedNodeBalancerConfig = { ...nodeBalancerConfig, ...payload };
+      await mswDB.update(
+        'nodeBalancers',
+        nodeBalancerId,
```

Comment:
Updates persisted for me once I changed this to reference the config nodes.

```suggestion
        'nodeBalancerConfigNodes',
        configId,
```